lib/axiom/fs/base/execute_context.js

```diff
@@ -213,6 +205,27 @@ ExecuteContext.prototype.setStdio = function(stdio) {
 };
 
 /**
+ * @return {!Stdio}
+ */
+ExecuteContext.prototype.getStdio = function() {
```

Comment:
An alternative could be to leave just `this.stdio` and get rid of `this.stdin` etc. But the client code was using the latter in 99% of the cases, so I went with the minimal change for now.